charts/shared/common-gitops/Chart.yaml

```diff
@@ -25,4 +25,4 @@ name: common-gitops
 sources:
   - https://github.com/luminartech/helm-charts-public/charts/common-gitops
 type: library
-version: "1.0.3-1"
+version: "1.0.4"
```

Comment:
can we do `1.0.3-2`?
I think there are no significant changes that mandate the 1.0.4 patch rel.

Reply:
Of course, but:
1.  `-X` versions are treated as release candidates by helm and not returned by CLIs by default
2. There's no limitation in these numbers, i.e. there can be `1.0.99999`.
3. This PR is a small change, but major version is not changed either.
4. Public charts usually don't use `-X` versions.
5. It was decided to use `-X` versions in other charts managed by us in order to preserve the dependency chart's version, but `common-gitops`  chart does not have any dependencies.

Reply:
Makes sense.